pmg/api/v1.py

```diff
@@ -555,6 +571,7 @@ def committee_meeting_attendance_meetings_by_member():
 
 
 @api.route('/committee-meeting-attendance/data.xlsx')
+@api.route('/committee-meeting-attendance/data.xlsx/')
```

Comment:
Sure you want the ending slash? That looks weird to me.

Reply:
I had an issue earlier where it seemed to drop the query string, but it works.
So, yes, definitely dropping it.